int32 label
geometry_msgs/Pose pose
